src/posts/owa-eu-dma-submission-apple-ipados.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'OWA submits response to EU DMA investigation into Apple iPadOS'
+date: '2023-11-21'
+tags: ['Policy', 'Updates', 'EU']
+author: "Frances Berriman"
+---
+
+As the EU further rolls out the Digital Marketing Act, they have designated a variety of companies and their platforms as “Gatekeepers”. You can see the [complete list of Gatekeepers](https://ec.europa.eu/commission/presscorner/detail/en/ip_23_4328) and some really useful explainers on what being a [Gatekeeper means on the EU’s very informative website](https://commission.europa.eu/strategy-and-policy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en).
+
+Now that Gatekeepers have been designated, the companies that operate those services have had the opportunity to rebut their designations. 
+
+A particularly interesting response was from Apple, who have made the case that iPadOS is a separate operating system entirely from iOS, both of which are currently designated as platforms subject to the DMA, along with the Apple AppStore. Their arguments essentially boil down to usage statistics for iPadOS alone being below the required usage numbers to fall under the rules and should not be designated a Gatekeeper platform service.
+
+The [EU consequently opened an investigation into the merits of this claim](https://ec.europa.eu/competition/digital_markets_act/cases/202343/DMA_100047_5136.pdf), and as such, OWA had the opportunity to respond as to why we believe that iPadOS should not have the Gatekeeper designation removed for this platform, and should not be considered separate from iOS in general.  
+
+In broad strokes, OWA argued that:
+
+
+
+* Apple’s argument about the size of the EU iPad market is in dispute. [IDC](https://www.idc.com/getdoc.jsp?containerId=IDC_P36344), for example, shows the iPad market in Europe (the UK inclusive) to be nearly 45MM devices, far above Apple’s claimed active monthly user base. At minimum, Apple needs to show its work. 
+
+* The distinction between iOS and iPadOS is only skin deep.  
+ 
+For most of the history of the iPad, iOS was it’s Operating System, and the launch schedules and differences in feature sets have not substantially diverged [since the 2019 marketing change](https://en.wikipedia.org/wiki/IPadOS#History). 
```

Comment:
Nit pick: `Operating System` should be lowercase?

Reply:
Yeah. This makes me think we could do with a lightweight style guide, actually. We proper noun some things but not others, and not consistently.